### PR TITLE
npm help, npm main debug commands, etc

### PR DESCRIPTION
**Pull request by chrisnojima**
I was debugging some issues w/ the menu and needed the ability to debug the main process to work. this is kinda a big PITA so i made npm helpers, along with 'npm run help'

there's some small login/menu stuff but it doesn't really affect anything

@keybase/react-hackers

### CODE REVIEW

File: desktop/client.js [hunk 1]
```diff
@@ -15,7 +15,13 @@ http.get('http://localhost:4000/dist/main.bundle.js', function (response) {
   response.pipe(file)
   file.on('finish', function() {
     file.close(function () {
-      const e = spawn(electron, [name], {stdio: 'inherit'})
+      const params = [name]
+
+      if (process.env.USE_INSPECTOR) {
+        params.unshift('--debug-brk=5858')
```
Comment (MarcoPolo) [inline desktop/client.js]: What does this do?

File: shared/login/index.js [hunk 2]
```diff
@@ -16,26 +20,37 @@ export default class Login extends Component {
...
+      }
     }
 
     return {
       componentAtTop: {
         component: Login,
+        hideBack: true,
         props: {
-          formComponent: Form
+          formComponent: () => Form
```
Comment (MarcoPolo) [inline shared/login/index.js]: minor :sheep: if this is no longer a react class can we lowercase it? to just `form`

**Comment by MarcoPolo** [comment]
:+1:

File: desktop/client.js (hunk 1 above)
Comment (chrisnojima) [inline desktop/client.js]: This is to debug the main electron thread (the node one)

Comment (MarcoPolo) [inline desktop/client.js]: I mean the `--debug-brk`

Comment (chrisnojima) [inline desktop/client.js]: That's what I'm talking about. Look at the docs for node inspector. It
starts In a debugged state so you can put in break points to debug startup
stuff

https://www.npmjs.com/package/node-inspector

**Pull request merged**